Data/future/cme/margins/DC.csv

```diff
@@ -1,41 +1,75 @@
 date,initial,maintenance
-# https://www.cmegroup.com/tools-information/lookups/advisories/clearing/Chadv12-172.html
 20120425,1755,1300
-# https://www.cmegroup.com/tools-information/lookups/advisories/clearing/Chadv13-457.html
-20131003,1991,1475
-# https://www.cmegroup.com/tools-information/lookups/advisories/clearing/Chadv14-186.html
-20140509,2025,1500
-# https://www.cmegroup.com/tools-information/lookups/advisories/clearing/Chadv14-204.html
-20140527,1650,1500
-# https://www.cmegroup.com/tools-information/lookups/advisories/clearing/Chadv15-007.html
+20130102,550,500
+20130104,1430,1300
+20130116,550,500
+20130131,1430,1300
+20130213,550,500
+20130228,1430,1300
+20130320,550,500
+20130404,1430,1300
+20130417,550,500
+20130502,1430,1300
```

Comment:
This one still seems not quite correct. Perhaps can just skip all the mid-month ones.